server/src/main/java/org/jboss/as/server/ServerEnvironment.java

```diff
@@ -879,12 +879,8 @@ public File getBundlesDir() {
     }
 
     private void configureBundlesDir(String dirPath, Properties providedProperties) {
-        boolean haveDirProperty = dirPath != null;
         File tmp = getFileFromPath(dirPath);
         if (tmp == null) {
-            if (haveDirProperty) {
-                throw ServerLogger.ROOT_LOGGER.bundlesDirectoryDoesNotExist(new File(dirPath).getAbsoluteFile());
```

Comment:
I'm not sure about these changes. We should not lose this trace in case of calling this method with a null dirPath argument. And, in my opinion, although both methods are at the same class and are both private, we should not assume that if `getFileFromPath(dirPath)` returns null it means that `dirPath` was null.

If we remove this change, any possible future change on `getFileFromPath(dirPath)` would have a side effect on this one, and that can be inadvertent by the developer who is modifying the `getFileFromPath(dirPath)`

Reply:
@yersan ok, we keep it stay robust here. I'll close the PR.